ios/yaysornays/AppDelegate.m

```diff
@@ -47,6 +47,11 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
   rootViewController.view = rootView;
   self.window.rootViewController = rootViewController;
   [self.window makeKeyAndVisible];
+  UIStoryboard *launchScreenStoryboard = [UIStoryboard storyboardWithName:@"Launch Screen" bundle:nil];
```

Comment:
Ah here it is, so in the delegation you're pointing the storyboard at the right screen while launchScreenStoryboard is active?